bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -1262,7 +1262,7 @@ protected boolean waitForWritable(DistributionSchedule.WriteSet writeSet,
             final int maxBackoff = 4;
             final long deadline = startTime + TimeUnit.MILLISECONDS.toNanos(durationMs);
 
-            while (!isWriteSetWritable(writeSet, allowedNonWritableCount)) {
+            while (!(success = isWriteSetWritable(writeSet, allowedNonWritableCount))) {
```

Comment:
I think success can be named writableResult,like the code:
`while (!(writableResult = isWriteSetWritable(writeSet, allowedNonWritableCount))) {`